doc/cookbook/source/examples/evaluation/cross_validation.rst

```diff
@@ -0,0 +1,90 @@
+================
+Cross Validation
+================
+
+Cross validation allows to estimate an algorithm's performance on unseen data.
+
+It is based on training the model on a subset of the data, and testing on the
+remaining data.
+
+The process is repeated for several complementary partitions and and all validation
+results are combined, e.g. by averaging.
+
+One type of cross validation is :math:`k-`fold cross validation.
+
+Suppose we have the training data :math:`\mathbf{x}_1,\mathbf{x}_2, ..., \mathbf{x}_{N}`
+for a positive integer :math:`N`, then :math:`k-`fold cross validation partitions
+the data into :math:`P_1, P_2, ..., P_n` each with approximately equal size. Next, we train the model on
+:math:`P_1, P_2, ...,P_{n-1}` and test it on :math:`P_n` to get performance measure
+:math:`a_1`. Finally we train the model again from scratch on :math:`P_1, P_2, ...,P_{n-2}, P_{n}` and test it on :math:`P_{n-1}`
```

Comment:
I would remove the last examplek as you already said twice what is happening ("Finally we train the model again from scratch on :math:`P_1, P_2, ...,P_{n-2}, P_{n}` and test it on :math:`P_{n-1}`")

Reply:
@karlnapf  In the last sentence, I use ``a_1, a_2, ..., a_n`` so I think leaving this sentence would be important so as to avoid confusion at where ``a_2, ..., a_n`` came from?

Reply:
I think this is clear from the first two examples. If you disagree we can also leave it in.